test/controllers/api/v2/registration_tokens_controller_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class Api::V2::RegistrationTokensControllerTest < ActionController::TestCase
+  test 'user shall invalidate tokens for self' do
+    user = User.create :login => "foo", :mail => "foo@bar.com", :auth_source => auth_sources(:one)
+    FactoryBot.build(:jwt_secret, token: 'test_jwt_secret', user: user)
```

Comment:
You probably want to create the token as part of the user creation, and use the `create` method - to make sure the token is saved before the action.